protocol/x/prices/keeper/util_test.go

```diff
@@ -285,6 +297,12 @@ func TestIsCrossingIndexPrice(t *testing.T) {
 			indexPrice:     3,
 			expectedResult: false,
 		},
+		"Not Crossing: new = curr < index": {
```

Comment:
All crossing tests have 13 conditions to check - add missing cases here and below.